### ISSUE TITLE
Hi！after i built this,i met this error when i runtime

### ISSUE THREAD

**Issue by jlk1997** [yknishidate/vulkan-minimal-hybrid-rendering#3]
![image](https://github.com/yknishidate/vulkan-minimal-hybrid-rendering/assets/56379549/d4123997-31a0-4237-9c86-34f6b34855ed)
How  can i fix it?

**Comment by yknishidate** [comment]
Thank you! Your GPU driver doesn't support some extensions. Please update the driver.

**Comment by jlk1997** [comment]
but my GPU driver is the newest version.Do i need to do some other configuration after clone?

**Comment by yknishidate** [comment]
well... please tell me more info about GPU and driver? or run "vulkaninfo" and paste here the output.
I think you don't need any configuration.

**Comment by jlk1997** [comment]
my GPU is Rtx3060 ,and my GPU Driver is 535.98
![image](https://github.com/yknishidate/vulkan-minimal-hybrid-rendering/assets/56379549/d485117f-5a42-47ec-b43f-b9c475904d20)

**Comment by yknishidate** [comment]
This repo doesn't use DirectX, please install latest VulkanSDK.

**Comment by jlk1997** [comment]
yep, i have installed the latest VulkanSDK and configure in the project ,but its still have this error 

![image](https://github.com/yknishidate/vulkan-minimal-hybrid-rendering/assets/56379549/4f9f96ff-f1d8-45af-96e9-a819d07f61d6)

**Comment by yknishidate** [comment]
Sorry it is hard to understand the error since this repository omits checking for extensions.

Please use the SDK's vulkaninfo to check if the following 3 features are true.

```
VkPhysicalDeviceRayQueryFeaturesKHR:
------------------------------------
        rayQuery = true
        
VkPhysicalDeviceAccelerationStructureFeaturesKHR:
-------------------------------------------------
        accelerationStructure                                 = true
        accelerationStructureCaptureReplay                    = true
        accelerationStructureIndirectBuild                    = false
        accelerationStructureHostCommands                     = false
        descriptorBindingAccelerationStructureUpdateAfterBind = true
        
VkPhysicalDeviceDynamicRenderingFeatures:
-----------------------------------------
        dynamicRendering = true
```
